pingcap/tidb-binlog
# drainer/* add queryHistogram metrics and log slow sql

### What problem does this PR solve? 
add query histogram and warn log the slow sql

### What is changed and how it works?


### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
add dashboard and check it

Code changes

 Side effects

Related changes

- Need to update the `tidb-ansible` repository
